### PR TITLE
Detect invoking dir for templates

### PR DESCRIPTION
**Pull request by timqian**
When users execute `sls invoke` or `sls invoke local` in a template folder, try to detect `scf`/`multi-scf` instance directory.

### Related ticket
https://app.asana.com/0/1200011502754281/1200578828969029

### CODE REVIEW

File: src/cli/commands-cn/utils.js [hunk 1]
```diff
@@ -210,6 +210,66 @@ const loadInstanceCredentials = () => {
...
+        instanceInfo.functions = Object.keys(instanceYaml.inputs.functions);
+      }
+      instances.push(instanceInfo);
+    }
+  }
+
+  const multiScfInstances = instances.filter((instance) => instance.component === 'multi-scf');
+  const scfInstances = instances.filter((instance) => instance.component === 'scf');
+  if (scfInstances.length + multiScfInstances.length === 0) {
+    throw new Error('没有找到可执行的函数目录，请使用 --target 指定或检查后再试');
```
Comment (zongUMR) [inline src/cli/commands-cn/utils.js:248]: Should we use warnings instead of erro here and elsewhere?

Comment (timqian) [reply src/cli/commands-cn/utils.js:248]: The error thrown here will be caught here: https://github.com/serverless/components/pull/998/files#diff-2a4f4f9def1700b7cc7cf1d95bf2250063ea626cf3bc4a69101cc51cefbe796fR59 and users will see warnings.

**Review by zongUMR — APPROVED** [review]

**Comment by zongUMR** [comment]
Does `logs` command need this feature?

**Comment by timqian** [comment]
@zongUMR This PR is only for `invoke` and `invoke local` which are only related to SCF-related components. We can discuss later if we want to add similar behavior to commands like `logs`, `dev`, and other general commands.

**Pull request merged**